line/index.js

```diff
@@ -61,7 +61,7 @@ router.post('/', auth0Middleware(), async (req, res) => {
     let newLine
     // This feels like a use case for /bulkCreate in RERUM.  Make all these lines with one call.
     for (const lineData of inputLines) {
-      newLine = Line.build(req.params.projectId, req.params.pageId, { ...lineData })
+      newLine = Line.build(req.params.projectId, req.params.pageId, { ...lineData }, user._id)
```

Comment:
creator isn't the req.user._id it is the agent from that token. It may be handy in the object, but can also be assembled without trouble as we worked hard to make the hexcodes match.